src/mon/PGMap.cc

```diff
@@ -2527,16 +2528,21 @@ void PGMap::get_health_checks(
         summary = "Degraded data redundancy: ";
         sev = HEALTH_WARN;
         break;
-      case DEGRADED_FULL:
-        health_code = "PG_DEGRADED_FULL";
-        summary = "Degraded data redundancy (low space): ";
-        sev = HEALTH_ERR;
+      case BACKFILL_FULL:
+        health_code = "PG_BACKFILL_FULL";
+        summary = "Low space blocking backfill (add storage if this doesn't resolve itself): ";
```

Comment:
maybe ``Low space blocking data migration (add storage if this doesn't resolve itself)``? we should avoid 'backfill' in the untrained-operator-facing health message if we can

Reply:
Actually, let's just do ``Low space blocking data migration``.  They can read the doc for the warning to tell that it may resolve itself (like many other warning conditions).

Reply:
@liewegas How about `Low space hindering data migration`?  _Blocking_ feels a little strong when it could become resolved all by itself.

Reply:
I like 'hindering'.  I'm also thinking we can drop the parenthetical "(add storage if this doesn't resolve itself)" -- they user can go look in the docs to learn about how to mitigate warnings/errors.

Reply:
hindering sounds good. I like having the mitigation in the warning itself, so users don't always have to look up docs for things with clear fixes though

Reply:
@liewegas @jdurgin To prevent actual out of space conditions the code overestimates space requirements. A backfill_toofull condition can therefore appear in non-space constrained situations.  Because of this I want to inform the user that the situation may resolve.